audioread/ffdec.py

```diff
@@ -87,7 +87,7 @@ def popen_multiple(commands, command_args, *args, **kwargs):
         cmd = [command] + command_args
         try:
             return subprocess.Popen(cmd, *args, **kwargs)
-        except OSError:
+        except (OSError, subprocess.CalledProcessError):
```

Comment:
While this seems pretty straightforward, it's actually a subtle change in behavior that might be undesirable. This `popen_multiple` function is meant to try multiple commands until it finds one that *exists*, i.e., to let us try both the `ffmpeg` and `avconv` names for the tool we want. It suppresses `OSError`s because those indicate that one of the binaries doesn't exist, which is pretty normal. Suppressing `CalledProcessError`s would do something different: it would hide any underlying error emitted by the program, such as when then audio file itself does not exist or is in the wrong format. I don't think we want to do that—these errors need to get handled differently by client code.

Reply:
Yeah, your right.
I wanted to do a bunch of changes to the ffdec.py module, regarding the change to subprocess.run and a integrated check of the process. The problem of this was that all these cool features are not backwards compatible to the 2.X releases. This check is a leftover of these changes and I share your concerns.